doc/amazon_sagemaker_debugger.rst

```diff
@@ -0,0 +1,377 @@
+    from smdebug_rulesconfig import vanishing_gradient, weight_update_ratio
+
+    estimator = TensorFlow(
+            role=role,
+            train_instance_count=1,
+            train_instance_type=train_instance_type,
+            rules=[Rule.sagemaker(vanishing_gradient()), Rule.sagemaker(weight_update_ratio())]
+    )
+
+In the example above, SageMaker pulls the hook configurations for Vanishing Gradient and Weight Update Ratio rules from `SageMaker Debugger Rules Config <https://github.com/awslabs/sagemaker-debugger-rulesconfig>`__  and configures the collections to be stored in the manner specified in each configuration.
+
+**Example 3**: Using a built-in rule with no customization and another built-in rule with customization.
+
+Here we modify the ``weight_update_ratio`` rule to store a custom collection rather than "weights" which it would normally do if the behavior is not overridden.
+
+
+.. code:: python
+
+    from sagemaker.debugger import Rule
+    from smdebug_rulesconfig import vanishing_gradient, weight_update_ratio
+
+    wur_with_customization = Rule.sagemaker(
+        base_config=weight_update_ratio(),
+        name="custom_wup_rule_name",
+        rule_parameters={
+            'key1': 'value1',
+            'key2': 'value2'
+        },
+        collections_to_save=[
+            CollectionConfig(
+                name="custom_collection_name",
+                parameters= {
+                    'key1': 'value1',
+                    'key2': 'value2'
+                }
+            )
+        ]
+    )
+
+    estimator = TensorFlow(
+            role=role,
+            train_instance_count=1,
+            train_instance_type=train_instance_type,
+            rules=[
+                Rule.sagemaker(vanishing_gradient()),
+                wur_with_customization
+            ]
+    )
+
+
+In the example above, collection configuration for Vanishing Gradient is pulled from `SageMaker Debugger Rules Config <https://github.com/awslabs/sagemaker-debugger-rulesconfig>`__  and the user supplied configuration is used for the Weight Update Ratio rule.
+
+Using custom rules
+------------------
+
+SageMaker Debugger also allows the users to create custom rules and have those evaluated against the debugging data. To use custom rules there are two prerequisites:
+
+* Custom rule source file and its local or S3 location. You can learn more about how to write custom rules at `How to Write Custom Debugger Rules <https://github.com/awslabs/sagemaker-debugger/blob/master/docs/analysis.md#writing-a-custom-rule>`__
+* Rule evaluator image for the corresponding region available from `Amazon SageMaker Debugger Custom Rule Images <https://docs.aws.amazon.com/sagemaker/latest/dg/debuger-custom-rule-registry-ids.html>`__
+
+To learn more about how to write your custom rules and use them see `SageMaker Debugger Custom Rules <https://docs.aws.amazon.com/sagemaker/latest/dg/debugger-custom-rules.html>`__
+
+Sample Usage
```

Comment:
would it make sense to link to the [full notebook](https://github.com/awslabs/amazon-sagemaker-examples/tree/master/sagemaker-debugger/tensorflow_keras_custom_rule) in this section?

Reply:
The debugger page which it currently links to does point to the notebook examples. It makes sense to point to the main documentation page rather than the example.

Reply:
"To use custom rules there are two prerequisites" -> "To use custom rules, you must provide two items:"

Reply:
@aws-rbs Corrected.